### ISSUE TITLE
[Bug]: Nextcloud 28 does not ask for filename when creating a new file

### ISSUE THREAD

**Issue by arnowelzel** [nextcloud/server#42483]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Before Nextcloud 28, when creating a new file in the web UI, you asked for the name for the file.

Since Nextcloud 28 files will only created using a standard name like `New text file.md`, which is confusing since you don't know where your new file is. Since the file may also not be visible within the current viewport, it can be become quite cumbersome in folders with many files where `New text file.md` is more towards the end of the files list. But without knowing where it is, you can also not open or rename it easily without scrolling up or down to find it.

### Steps to reproduce

1. Open the files app
2. Use the "+ New" and select "New Textfile"
3. Select a template if there are templates defined - but it also is reproducable without templates
4. See the file being created as `New text file.md` without a chance to enter the name first

### Expected behavior

Nextcloud will ask for a filename before creating it.

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

28

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Upgraded to a MAJOR version (ex. 22 to 23)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "skeletondirectory": "",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "28.0.1.1",
        "installed": true,
        "htaccess.RewriteBase": "\/",
        "maintenance": false,
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "filelocking.enabled": true,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "dbindex": 0
        },
        "knowledgebaseenabled": false,
        "theme": "",
        "simpleSignUpLink.shown": false,
        "loglevel": 2,
        "log_authfailip": true,
        "mysql.utf8mb4": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "sendmail",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "trashbin_retention_obligation": "auto, 30",
        "versions_retention_obligation": "auto, 180",
        "app_install_overwrite": [
            "apporder",
            "deck",
            "keeweb",
            "appointments",
            "drawio",
            "bookmarks",
            "spreed",
            "unroundedcorners",
            "fulltextsearch",
            "fulltextsearch_elasticsearch",
            "files_fulltextsearch",
            "files_fulltextsearch_tesseract",
            "passwords",
            "gpxpod",
            "dicomviewer",
            "files_bpm"
        ],
        "has_rebuilt_cache": true,
        "default_language": "de",
        "default_phone_region": "DE",
        "check_data_directory_permissions": false,
        "log_rotate_size": 52428800,
        "profile.enabled": false,
        "preview_max_x": 1920,
        "preview_max_y": 1920,
        "mail_sendmailmode": "smtp",
        "mail_smtpauth": 1,
        "mail_smtpport": "465",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "auth.bruteforce.protection.enabled": true,
        "ratelimit.protection.enabled": true,
        "reference_opengraph": false,
        "integrity.check.disabled": false
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.20.0
  - admin_audit: 1.18.0
  - bruteforcesettings: 2.8.0
  - calendar: 4.6.1
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - collectives: 2.9.2
  - comments: 1.18.0
  - contacts: 5.5.0
  - contactsinteraction: 1.9.0
  - dav: 1.29.1
  - deck: 1.12.0
  - dicomviewer: 1.2.5
  - external: 5.3.1
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_bpm: 1.2.0
  - files_fulltextsearch: 28.0.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - fulltextsearch: 28.0.0
  - fulltextsearch_elasticsearch: 28.0.0
  - impersonate: 1.15.0
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - nextcloud_announcements: 1.17.0
  - notes: 4.9.1
  - notifications: 2.16.0
  - notify_push: 0.6.6
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - photos: 2.4.0
  - previewgenerator: 5.4.0
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - richdocuments: 8.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - support: 1.11.0
  - survey_client: 1.16.0
  - systemtags: 1.18.0
  - tasks: 0.15.0
  - text: 3.9.1
  - theming: 2.3.0
  - theming_customcss: 1.15.0
  - twofactor_backupcodes: 1.17.0
  - twofactor_webauthn: 1.3.2
  - updatenotification: 1.18.0
  - viewer: 2.2.0
  - workflowengine: 2.10.0
Disabled:
  - dashboard: 7.8.0 (installed 7.7.0)
  - encryption: 2.16.0 (installed 2.13.0)
  - files_external: 1.20.0 (installed 1.19.0)
  - recommendations: 2.0.0 (installed 0.4.0)
  - related_resources: 1.3.0 (installed 1.1.0-alpha1)
  - suspicious_login: 6.0.0
  - twofactor_totp: 10.0.0-beta.2
  - user_ldap: 1.19.0
  - user_status: 1.8.1 (installed 1.5.0)
  - weather_status: 1.8.0 (installed 1.3.0)
```

### Nextcloud Signing status

```shell
No errors have been found.
```

**Comment by joshtrichards** [comment]
This may have gotten overlooked after #40854 was merged during the great f2v migration.

Looks like it came up here: https://github.com/nextcloud/server/pull/40854#issuecomment-1759176240

**Comment by joshtrichards** [comment]
Cc: @skjnldsv

**Comment by skjnldsv** [comment]
@joshtrichards stop pinging me directly please. labels are set, you can wait for anyone to pick that issue up.

**Comment by solracsf** [comment]
Does https://github.com/nextcloud/server/pull/42443 fix this?

**Comment by skjnldsv** [comment]
@solracsf kind of :)
But there is still another discussion on the overall changed process for 28.
See https://github.com/nextcloud/server/issues/41260 and https://github.com/nextcloud/server/issues/42097

**Comment by rchaconmolero** [comment]
Upgrading form 28.0.1 to  Nextcloud 28.0.2 RC1 the problem persists. ¿Is it normal?. Thanks.

**Comment by arnowelzel** [comment]
> Upgrading form 28.0.1 to Nextcloud 28.0.2 RC1 the problem persists. ¿Is it normal?. Thanks.

Since RC is "Release Candidate" and not "Release" - yes, this may be normal, if the change was not yet merged to the branch of Version 28.

**Comment by rchaconmolero** [comment]
Will this "problem" be resolved in the new release, 28.0.3 or is it still in the study phase? The truth is, the way it has always worked is great. Thanks.

**Comment by mehlkopf** [comment]
Unfortunately, the "stable" 28.0.2 is out and the problem persists. As just written on another files app bug, this version 28 cycle is really frustrating.

**Comment by bentuna** [comment]
We can confirm that this issue persists with 28.0.2 when creating new **files**. 

When creating a new folder, the issue seems to be solved though.

**Comment by Luncheon3462** [comment]
Who is able to change the status from closed.

**Comment by arnowelzel** [comment]
> Who is able to change the status from closed. 

This is not needed. There is another issue #42097 which will address this.

**Comment by Luncheon3462** [comment]
28.0.3 While I can create a file name, the document does not open.   Correct behavior is creating a file name and then the document should open automatically.   NC27 works fine/appropriately.
